src/frontends/tensorflow/src/op/approximateequal.cpp

```diff
@@ -0,0 +1,40 @@
+// approximateequal.cpp
+#include <ngraph/node.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/validation_util.hpp>
```

Comment:
please avoid use of `ngraph`, this release we tootally remove it.
Just see this PR as an example https://github.com/openvinotoolkit/openvino/pull/19007 where we use only ov headers